pkg/util/util_test.go

```diff
@@ -416,3 +416,53 @@ func TestRestorePossible(t *testing.T) {
 		})
 	}
 }
+
+func TestIsMacAddressCleared(t *testing.T) {
+	testCases := []struct {
+		name     string
+		resource string
+		vm       kvcore.VirtualMachine
+		expected bool
+	}{
+		{"Clear mac addres should return false",
```

Comment:
the name of the tests should also explain what is tested i.e here: with no annotation, in the next with annotation false etc..